LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2017 Infinite Red, http://infinite.red
```

Comment:
Update made.

I noticed the README uses the non-https URL and also leaves the Inc. off of the company name if you want either of these updated for consistency.